storage/postgres/client.go

```diff
@@ -250,9 +250,32 @@ func (c *Client) Wipe(ctx context.Context) error {
 		}
 	}
 
+	// list, then drop some extensions.
+	rows, err := c.Query(ctx, `
+		SELECT extname
+		FROM pg_extension
+	`)
+	if err != nil {
+		return fmt.Errorf("failed to list extensions: %w", err)
+	}
+	defer rows.Close()
+	for rows.Next() {
+		var extension string
+		if err = rows.Scan(&extension); err != nil {
+			return err
+		}
+		// This one was here when I got here.
+		if extension == "plpgsql" {
```

Comment:
never heard of this extension before, so not touching it??? :skull: :skull: :skull: :skull:

Reply:
I'm surprised this is even an extension, technically. It's the de-facto default procedural language for writing functions in postgres. And yes, good call not to remove, I've used it in ad-hoc ways before, and we might use it later.

WDYT about avoiding this `if` and just doing `SELECT ... WHERE extname != 'plpgsql'` instead? It's how most other filtering in Wipe() is done.

Reply:
oh ok

although let's go with the create extension if not exists